client/task-list/tasks/payments/methods/index.js

```diff
@@ -342,7 +341,7 @@ export function getPaymentMethods( {
 		{
 			key: 'mollie',
 			title: __( 'Mollie Payments for WooCommerce', 'woocommerce-admin' ),
-			before: <MollieLogo />,
+			before: <img src={ wcAdminAssetUrl + 'mollie.svg' } alt="" />,
```

Comment:
Good call updating this 👍

Reply:
Its weird the other images never had alts either , but it totally should :/ I'll add one

Reply:
I think we should be translating the alt too? I'm gonna add a translation call here, but yeah the other images will still be missing them. It seems out of scope to fix that here, but I could open another PR if its desired, seems maybe low value if this is going away.

Reply:
Resolved here https://github.com/woocommerce/woocommerce-admin/pull/7092/commits/809c8aa6c26c457f59b2b08887bc2f765376a5d7